.github/workflows/update-snapcraft.yaml

```diff
@@ -0,0 +1,44 @@
+on:
+  workflow_call:
+    inputs:
+      openstack-release:
+        description: OpenStack release
+        type: string
+        required: true
+
+jobs:
+  update-snapcraft:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: stable/${{ inputs.openstack-release }}
+      - name: Install Workflow Dependencies
+        run: pip install -r requirements-dev.txt
```

Comment:
planning to impement the tox target idea. and thinking hardcoding the dependencies right inside the `tox.ini` rather than a separate requirements file. I don't like the idea of maintaining short files here and there. objections?

Reply:
I was also thinking to use only dependencies in `tox.ini`, so :+1:  from me.

Reply:
implemented in update